### PR TITLE
Document fixes

### PR DESCRIPTION
**Pull request by eleuzi01**
- fixed expired image link in mbedtls-tutorial.md
- changed title headers in reduce-polarssl-memory-and-storage-footprint.md to maintain consistency in readthedocs sidebar

### CODE REVIEW

File: kb/how-to/mbedtls-tutorial.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Mbed TLS is designed in the portable C language with embedded environments as a
 
 The aim of this tutorial is to show you how to secure your client and server communication with Mbed TLS. The following major components  are involved:
 
-![Mbed TLS Stack.png](https://mbed-tls-docs-images.s3.amazonaws.com/mbedtls-stack.png)
+![Mbed TLS Stack.png](https://web.archive.org/web/20210921191453/https://mbed-tls-docs-images.s3.amazonaws.com/mbedtls-stack.png)
```
[the remark this answers is not in the document]
Comment (paul-elliott-arm) [reply kb/how-to/mbedtls-tutorial.md:13]: I'm unsure of the reason it was in an S3 bucket in the first place tbh - @bensze01 was this you?

File: kb/how-to/reduce-polarssl-memory-and-storage-footprint.md [hunk 2]
```diff
@@ -6,15 +6,15 @@ All of the settings described on this page are available in `mbedtls_config.h`,
...
 
 The binary footprint is the size of the actual file on disk, in the ROM or the flash.
 
-## Minimizing features
+### Minimizing features
 
 By default, Mbed TLS offers several compatibility options and frequently used functionalities. To reduce the footprint, adapt `mbedtls_config.h` to disable the functions that you do not need.
 
-# Memory footprint
+## Memory footprint
```
Comment (gilles-peskine-arm) [inline kb/how-to/reduce-polarssl-memory-and-storage-footprint.md:17]: Please downgrade the subsections and subsubsections as well.

**Review by gilles-peskine-arm — CHANGES_REQUESTED** [review]
The title level change is incomplete.

**Review by gilles-peskine-arm — APPROVED** [review]
LGTM

**Pull request merged**